pupil-spa/src/app/question/question.component.ts

```diff
@@ -11,13 +11,26 @@ import { WindowRefService } from '../services/window-ref/window-ref.service';
 })
 export class QuestionComponent extends PracticeQuestionComponent implements OnInit, AfterViewInit {
 
+  /**
+   * Do not show 'practice' label on top left.
+   */
+  public isWarmUpQuestion: boolean;
+
   constructor(protected auditService: AuditService,
               protected windowRefService: WindowRefService,
               protected registerInputService: RegisterInputService) {
     super(auditService, windowRefService);
     this.window = windowRefService.nativeWindow;
   }
 
+  ngOnInit() {
+    this.remainingTime = this.questionTimeoutSecs;
+  }
+
+  ngAfterViewInit() {
+    this.isWarmUpQuestion = false;
```

Comment:
Why not initialise this in the declaration?

E..g

`public isWarmupQuestion = false`

It's always false isn't it for this class?

Reply:
@jon-shipley That was my reasoning too however, it was set to true in the warm-up questions and it seems to remain true unless I explicitly change it to false in the check questions.

Reply:
It feels wrong to set the variable after the view has been rendered.  It would be better in the constructor then.

Reply:
@jon-shipley Done.